examples/hotrod/docker-compose.yml

```diff
@@ -12,13 +9,13 @@ services:
       - "4318:4318"
     environment:
       - LOG_LEVEL=debug
+      - COLLECTOR_OTLP_GRPC_HOST_PORT=0.0.0.0:4317
+      - COLLECTOR_OTLP_HTTP_HOST_PORT=0.0.0.0:4318
     networks:
       - jaeger-example
+
   hotrod:
-    image: jaegertracing/example-hotrod:${JAEGER_VERSION:-latest}
-    # To run the latest trunk build, find the tag at Docker Hub and use the line below
-    # https://hub.docker.com/r/jaegertracing/example-hotrod-snapshot/tags
-    #image: jaegertracing/example-hotrod-snapshot:0ab8f2fcb12ff0d10830c1ee3bb52b745522db6c
+    image: ${HOTROD_IMAGE:-jaegertracing/example-hotrod}:${GITHUB_SHA:-${JAEGER_VERSION:-latest}}
```

Comment:
GITHUB_SHA has nothing to do with this script. JAEGER_VERSION variable is sufficient.

Reply:
> HOTROD_IMAGE:-jaegertracing/example-hotrod

Are you actually planning to override the name, or just the registry? I think it's only the registry, so specialize the var to that

```
image: ${REGISTRY:-}jaegertracing/example-hotrod:${JAEGER_VERSION:-latest}
```

Reply:
> GITHUB_SHA has nothing to do with this script. JAEGER_VERSION variable is sufficient.

but the local image build by `scripts/build-upload-a-docker-image.sh` gives it a tag like this so that is why I need GITHUB_SHA
`"localhost:5000/${namespace}/${component_name}:${GITHUB_SHA}"`

Reply:
It doesn't matter if another script needs it, that should be taken care of as an integration point. This docker-compose itself has no need for GITHUB_SHA, it only needs JAEGER_VERSION

Reply:
but the local image that is created for testing  has name 
localhost:5000/jaegertracing/example-hotrod:${GITHUB_SHA}
so I want to test using this image I need to use :${GITHUB_SHA} in docker compose otherwise it will pull the latest image instead of using the local build binary

Reply:
So call the compose file with `JAEGER_VERSION=${GITHUB_SHA}` env var.

Reply:
but then it will also use it for ` jaegertracing/all-in-one:${JAEGER_VERSION:-latest}` which will give error